src/pugixml.cpp

```diff
@@ -6059,6 +6059,23 @@ namespace pugi
 		return true;
 	}
 
+	PUGI__FN bool xml_node::remove_attributes()
```

Comment:
Note, I've left a few comments in remove_children, most/all of them apply here as well.